Gemfile

```diff
@@ -40,17 +40,18 @@ group :development do
 end
 
 group :system_tests do
-  gem 'winrm',                              :require => false
-  if beaker_version = ENV['BEAKER_VERSION']
-    gem 'beaker', *location_for(beaker_version)
-  else
-    gem 'beaker', '>= 3.9.0', :require => false
-  end
-  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
-    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  gem 'pry'
+  gem 'pry-stack_explorer'
+  if RUBY_VERSION =~ /^2/
+    gem 'pry-byebug'
   else
-    gem 'beaker-rspec',  :require => false
+    gem 'pry-debugger'
   end
+  gem 'winrm',                              :require => false
+  gem 'beaker',                             :require => false
+  gem 'beaker-puppet',                      :require => false
+  gem 'beaker-vagrant',                     :require => false
```

Comment:
`beaker-vagrant` needs to be added to the `.sync.yml` file. An example can be found here:
https://github.com/voxpupuli/puppet-grafana/blob/79745359d28fc476fc2aab30970df442a43e6463/.sync.yml#L7-L10

Reply:
done